src/main/java/kitchenpos/application/ProductService.java

```diff
@@ -18,14 +19,8 @@ public ProductService(final ProductDao productDao) {
     }
 
     @Transactional
-    public Product create(final Product product) {
-        final BigDecimal price = product.getPrice();
-
-        if (Objects.isNull(price) || price.compareTo(BigDecimal.ZERO) < 0) {
-            throw new IllegalArgumentException();
-        }
-
-        return productDao.save(product);
+    public Product create(final ProductRequest request) {
+        return productDao.save(request.toEntity());
```

Comment:
취향일 것 같은데요,
```suggestion
        return productDao.save(request.toProduct());
```
형식으로 사용하는게 더 좋아보이더라구요 :)
참고만 부탁드리겠습니다 🙏

Reply:
저도 이 방법이 훨씬 가독성이 좋은 것 같습니다! 좋은 팁 감사합니다 🙇‍♂️